assets/javascripts/discourse/templates/modal/invalid-codebyte-modal.hbs

```diff
@@ -0,0 +1,10 @@
+{{#d-modal-body title="modal.title"}}
+  <div>{{i18n "modal.content"}}</div>
+  <div>
+    {{d-button
+      action=(action "goBackAndFix")
+      label="modal.button"
+      class="btn-primary"
+    }}
+  </div>
+{{/d-modal-body}}
```

Comment:
<img width="705" alt="Screen Shot 2021-06-14 at 12 15 56 PM" src="https://user-images.githubusercontent.com/17210163/121925037-7f0f2c80-cd0a-11eb-9304-5bd3efd887cf.png">

Just caught this lil CSS issue with the hover state on this close button.

Reply:
Potential fix quoted from my review comment here: https://github.com/Codecademy/discourse-codebytes-plugin/pull/37#pullrequestreview-683108807


> ```css-scss
> // assets/stylesheets/common/code-bytes.scss
> .codebytes-invalid-modal .modal-header .modal-close {
>   padding-left: 0;
> }
> ```

Reply:
Huh, I'm not able to reproduce that. What browser and screen size are you on? I wonder if it's because of the theme components that I have installed on my local setup.
I'll add in the css fix just in case though.

Reply:
I'm on Chrome, `Version 91.0.4472.101`, and I can reproduce on any screen size.
Also good point on the theme, my setup hasn't changed since: https://github.com/Codecademy/discourse-codebytes-plugin/pull/24/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R129-R133

Reply:
After release to prod, we can remove that rule in DevTools to see if that CSS fix was needed or not?